commit bfcc0d204db7fba5e6bdbe062116627f0b4f5b34 (refs/changes/07/4207/2)
Author: Orlando Vazquez <ovazquez@gmail.com>
Date:   2018-06-13T14:30:39-07:00 (1 year, 4 months ago)
    
    TRITON-503 CNAPI sendTaskRequest not asserting the req_id it ends up using

diff --git a/lib/models/server.js b/lib/models/server.js
index 7dffbd3..ccabe5c 100644
--- a/lib/models/server.js
+++ b/lib/models/server.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright (c) 2017, Joyent, Inc.
+ * Copyright (c) 2018, Joyent, Inc.
  */
 
 /*
@@ -2115,7 +2115,7 @@ function (opts) {
 
     var taskstatus = {
         id: common.genId(),
-        req_id: opts.req_id,
+        req_id: req_id,
         task: opts.task,
         server_uuid: self.uuid,
         status: 'active',
